src/config.js

```diff
@@ -21,7 +21,14 @@ let defaults = {
   DB_CERT: undefined,
   // factoid specific urls
   FACTOID_URL: 'http://unstable.factoid.baderlab.org/',
-  BIOPAX_CONVERTERS_URL: 'http://biopax.baderlab.org/'
+  BIOPAX_CONVERTERS_URL: 'http://biopax.baderlab.org/',
+  DATASOURCE_NAMES :{// !!!temporary , will be updated as part of 'identifiers url generation module' https://github.com/PathwayCommons/app-ui/issues/1116
```

Comment:
You can't have embedded objects with `config`.  An env var can't specify an object -- only a string value.

Reply:
Why not just use a consistent value instead?

E.g. just `HGNC_SYMBOL` -- never `HGNCSYMBOL`.